docs/translation.md

```diff
@@ -0,0 +1,52 @@
+# Translation Guidelines
+
+Thank you for your interest in contributing translations to Keycloak! 
+This document provides guidelines for contributing translations and getting started with Weblate.
+
+## Why Translation Matters
+
+Translation enables Keycloak to reach a wider audience by making the platform usable for speakers of various languages. By translating Keycloak, you help improve accessibility and usability for users who may not be proficient in English.
+
+## What to translate
+
+Your contributions can greatly enhance the user experience for non-English speakers.
+
+Keycloak welcomes translations of its user interfaces and error messages via Weblate. 
+
+There is also a [Keycloak Documentation translation project](https://github.com/openstandia/keycloak-documentation-i18n). 
```

Comment:
Perhaps good to mention that this is not maintained by the Keycloak team itself, but a third party.